vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/DocumentOperationExecutor.java

```diff
@@ -52,8 +52,9 @@ default void shutdown() { }
         NOT_FOUND,
         PRECONDITION_FAILED,
         BAD_REQUEST,
+        ERROR,
```

Comment:
Reordering an enum should in practice not be a problem, but any usage of `.ordinal()` may result in unexpected behaviour.

Reply:
Which is why the use of `ordinal()` is discouraged, and definitely not used for this (brand new) enum :)

Reply:
Both `.name()` and `.ordinal()` is bad practice, yet we do it (at least the former) in our own code ;)